auth-web/src/components/auth/mixins/DateMixin.vue

```diff
@@ -0,0 +1,28 @@
+// You can declare a mixin as the same style as components.
+<script lang="ts">
+import Component from 'vue-class-component'
+import Vue from 'vue'
+
+@Component({
+  name: 'DateMixin'
```

Comment:
Are you sure this is needed?

Reply:
This syntax? It's how the rest of the mixins are declared in this repo, so i followed suite. 

If you are referring to the mixin itself, yes, this date formatting method could/would be re used i imagine.

Reply:
Yes, I meant the syntax.